web/wp-content/plugins/tmbi-adobe-dtm/legacy/single_data_layer.php

```diff
@@ -10,15 +10,31 @@ function dtm_add_single_data( $data_layer ) {
 	if ( is_single() ) {
 		global $post, $numpages;
 
-		$source = apply_filters( 'dtm_data_layer_page_content_source', false );
+		if ( is_page( 'fbia-dax' ) && ! empty( $_GET['contentId'] ) && $_GET['contentId'] > 0  ) {
+			$post = get_post( intval( $_GET['contentId'] ) );
+			setup_postdata( $post );
```

Comment:
This sets global variables (`$id, $authordata, $currentday, $currentmonth, $page, $pages, $multipage, $more, $numpages`) but none of them are needed in this function. Why is it needed?